doc/source/tune/examples/tune-xgboost.ipynb

```diff
@@ -1,1256 +1,1256 @@
-                "into 4 and 96 samples, respectively. In the next step, our model might find\n",
-                "that it doesn't make sense to split the 4 examples more. It thus only continues\n",
-                "to add leaves on the right side.\n",
-                ":::\n",
-                "\n",
-                "The parameter used by the model to decide if it makes sense to split a node is called\n",
-                "the *minimum child weight*. In the case of linear regression, this is just the absolute\n",
-                "number of nodes requried in each child. In other objectives, this value is determined\n",
-                "using the weights of the examples, hence the name.\n",
-                "\n",
-                "The larger the value, the more constrained the trees are and the less deep they will be.\n",
-                "This parameter thus also affects the model complexity. Values can range between 0\n",
-                "and infinity and are dependent on the sample size. For our ca. 500 examples in the\n",
-                "breast cancer dataset, values between `0` and `10` should be sensible.\n",
-                "\n",
-                "XGBoost's default value is `1`.\n",
-                "\n",
-                "### Subsample size\n",
-                "\n",
-                "Each decision tree we add is trained on a subsample of the total training dataset.\n",
-                "The probabilities for the samples are weighted according to the XGBoost algorithm,\n",
-                "but we can decide on which fraction of the samples we want to train each decision\n",
-                "tree on.\n",
-                "\n",
-                "Setting this value to `0.7` would mean that we randomly sample `70%` of the\n",
-                "training dataset before each training iteration.\n",
-                "\n",
-                "XGBoost's default value is `1`.\n",
-                "\n",
-                "### Learning rate / Eta\n",
-                "\n",
-                "Remember that XGBoost sequentially trains many decision trees, and that later trees\n",
-                "are more likely trained on data that has been misclassified by prior trees. In effect\n",
-                "this means that earlier trees make decisions for easy samples (i.e. those samples that\n",
-                "can easily be classified) and later trees make decisions for harder samples. It is then\n",
-                "sensible to assume that the later trees are less accurate than earlier trees.\n",
-                "\n",
-                "To address this fact, XGBoost uses a parameter called *Eta*, which is sometimes called\n",
-                "the *learning rate*. Don't confuse this with learning rates from gradient descent!\n",
-                "The original [paper on stochastic gradient boosting](https://jerryfriedman.su.domains/ftp/stobst.pdf)\n",
```

Comment:
I don't know why GH still can't display notebook diffs well, but I only changed the link to this paper, fyi